test/integration/podgc/podgc_test.go

```diff
@@ -111,9 +131,9 @@ func TestPodGcOrphanedPodsWithFinalizer(t *testing.T) {
 				t.Fatalf("Error: '%v' while updating pod info: '%v'", err, klog.KObj(pod))
 			}
 			_, cond := podutil.GetPodCondition(&pod.Status, v1.DisruptionTarget)
```

Comment:
nit: a single `cmp.Diff` on the status could cover this check and the one below.

Reply:
done